packages/leavittbook/src/demos/titanium-card-two-line-list-item/titanium-card-two-line-list-item-playground.ts

```diff
@@ -0,0 +1,153 @@
+/* playground-fold */
+import { css, html, LitElement } from 'lit';
+import { customElement } from 'lit/decorators.js';
+import { h1, p } from '@leavittsoftware/titanium-styles';
+
+/* playground-fold-end */
+import '@leavittsoftware/titanium-card/lib/titanium-card-two-line-list-item';
+import '@leavittsoftware/titanium-card';
```

Comment:
I think we want the imports that are supplementary to the demo folded, so move titanium-card and mwc-icon up into the fold